static/style.css

```diff
@@ -104,8 +104,13 @@ body{
     justify-content: flex-end;
 }
 
+.HiddenMenu ul {
+    display: flex;
+    justify-content: flex-end;
+
```

Comment:
```suggestion
```
It is not necessary, as the parent div of the list already justifies the entire list at the end of the line